integration-tests/src/test/java/org/zowe/apiml/integration/proxy/WebSocketProxyTest.java

```diff
@@ -159,7 +160,7 @@ void headers(String path) throws Exception {
                 @Nested
                 class ReturnError {
                     @ParameterizedTest(name = "WhenRoutingSession.Authentication.WhenValid.ReturnSuccess#headers {0}")
-                    @ValueSource(strings = {"/discoverableclient/ws/v1/bad", "/ws/v1/discoverableclient/bad"})
+                    @ValueSource(strings = {"/discoverableclient/ws/v1/bad"})
```

Comment:
Should this be moved to Endpoints too?

Reply:
As they are invalid, I decided not to. To prevent the confusion I keep there only valid ones.